lib/ruby_lsp/requests/support/rubocop_runner.rb

```diff
@@ -14,17 +15,22 @@ module Support
       class RuboCopRunner < RuboCop::Runner
         extend T::Sig
 
-        class ConfigurationError < StandardError; end
+        class ConfigurationError < StandardError
+        end
```

Comment:
This and other layout changes below seem like they are coming from Syntax Tree formatting and not RuboCop. Can we remove those from this PR?